crates/gitbutler-stack-api/src/stack_ext.rs

```diff
@@ -17,6 +17,7 @@ use gitbutler_repo::RepositoryExt;
 use gitbutler_stack::Stack;
 use gitbutler_stack::VirtualBranchesHandle;
 use gix::validate::reference::name_partial;
+use gix_utils::str::decompose;
```

Comment:
This should be `gix::utils::str::decompose` - I validated that it is exported, particularly since [`gix` was updated](https://github.com/gitbutlerapp/gitbutler/pull/5233).